Jenkinsfile

```diff
@@ -60,7 +60,7 @@ node('mongodb-2.4') {
     }
 
     stage("Set up content schema dependency") {
-      govuk.contentSchemaDependency()
+      govuk.contentSchemaDependency(env.SCHEMA_BRANCH)
```

Comment:
Yeah, it's safe to remove it entirely. The jenkinslib in govuk-puppet used to call `env.IS_SCHEMA_TEST` but it also uses `params` now.